caffe2/core/typeid.cc

```diff
@@ -41,9 +44,9 @@ string Demangle(const char* name) {
 
 string GetExceptionString(const std::exception& e) {
 #ifdef __GXX_RTTI
-  return Demangle(typeid(e).name()) + ": " + e.what();
+      return Demangle(typeid(e).name()) + ": " + e.what();
```

Comment:
Is the reindentation here clang-format's fault? I thought we were two space indents in the codebase.

Reply:
Nice catch